exdb/forms.py

```diff
@@ -73,16 +79,13 @@ def clean(self):
         if ex_type.needs_verification and self.cleaned_data.get('start_datetime') < self.when:
             raise ValidationError(ex_type.name + " events cannot happen in the past")
 
-        if not ex_type.needs_verification and not self.cleaned_data.get('attendance'):
+        if not ex_type.needs_verification and (not self.cleaned_data.get(
+                'attendance') or self.cleaned_data.get('attendance') < 1):
             raise ValidationError(ex_type.name + " events must have an attendance")
 
         if not ex_type.needs_verification and not self.cleaned_data.get('audience'):
             raise ValidationError(ex_type.name + " events must have an audience")
 
-        if not ex_type.needs_verification and self.cleaned_data.get(
-                'attendance') and self.cleaned_data.get('attendance') < 1:
-            raise ValidationError(ex_type.name + " events must have an attendance greater than 0")
-
         if ex_type.needs_verification and self.cleaned_data.get('attendance'):
             raise ValidationError(ex_type.name + " events cannot have an attendance")
```

Comment:
Perhaps there is a less redundant representation for this task. It seems that every if statement determines if a ValidationError is going to be raised. At a glance it seems that you could put these statements in a for loop which iterates over a list of condition, error message pairs. I believe in the long run it will be easier to maintain. You should probably make it a utility function as well because I foresee these clean functions needing similar behavior in the future.